CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Cargo
+
+Thank you for your interest in contributing to Cargo! Good places to
+start are this document, [ARCHITECTURE.md](ARCHITECTURE.md), which
+describes high-level structure of Cargo and [E-easy] bugs on the
+issue tracker.
+
+As a reminder, all contributors are expected to follow our [Code of Conduct].
+
+[E-easy]: https://github.com/rust-lang/cargo/labels/E-easy
```

Comment:
@carols10cents could you add some details about the issue label system we use for Cargo (like, what each label means)? It would be probably useful for new contributors, and it would for sure be useful for me, because I have no idea what labels I should apply when I file the issues :)